app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuDataCollectionRobot.kt

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@file:Suppress("TooManyFunctions")
+
+package org.mozilla.fenix.ui.robots
+
+/**
+ * Implementation of Robot Pattern for the settings Data Collection sub menu.
+ */
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.Visibility
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.espresso.matcher.ViewMatchers.withEffectiveVisibility
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withParent
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.hamcrest.CoreMatchers.allOf
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.assertIsEnabled
+import org.mozilla.fenix.helpers.click
+
+class SettingsSubMenuDataCollectionRobot {
+
+    fun verifyNavigationToolBarHeader() = assertNavigationToolBarHeader()
+
+    fun verifyDataCollectionOptions() = assertDataCollectionOptions()
+
+    fun verifyUsageAndTechnicalDataSwitchDefault() = assertUsageAndTechnicalDataSwitchDefault()
+
+    fun verifyMarketingDataSwitchDefault() = assertMarketingDataValueSwitchDefault()
+
+    fun verifyDataCollectionSubMenuItems() {
+        verifyDataCollectionOptions()
+        verifyUsageAndTechnicalDataSwitchDefault()
+        verifyMarketingDataSwitchDefault()
+    }
+
+    class Transition {
+        val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())!!
+        fun goBack(interact: SettingsRobot.() -> Unit): SettingsRobot.Transition {
+            mDevice.waitForIdle()
+            goBackButton().click()
+
+            SettingsRobot().interact()
+            return SettingsRobot.Transition()
+        }
+    }
+}
+
+private fun goBackButton() =

```

Comment:
This is being defined in each Robot. Could you try to add it to one of the Helpers that are already defined? 
See here: https://github.com/mozilla-mobile/fenix/tree/master/app/src/androidTest/java/org/mozilla/fenix/helpers

Reply:
ok . I will add.

Reply:
https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

Would this the accurate file in which I can add 

fun goBackButton() = 
   onVIew(withContentDescription("Navigate up");

Reply:
https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

Would this be the accurate file in which the below can be added?

fun goBackButton() = 
onView(withContentDescription("Navigate up")

Reply:
I would say so, let's double check with @AaronMT...

Reply:
ok

Reply:
Hi @isabelrios , as we are unsure about the location of the goBackButton() to be added in the helpers file. Excluding that I have made all other changes asked.

Reply:
Oh you can do it in the file I said before, we can check if that works there and how it looks like, then make a decision, if that sounds good.

Reply:
Ok I wi do that. But goBackButton() is present in all other tests too too apart from my work in settingsprivacytest. Shouldn't it can taken under a separate PR?

Reply:
Yeah, you are right. We can do it in a follow up and then change all tests, or start here, only modifying your tests, start using it in new ones and only change the existing tests in a follow up. So, as you prefer, is not mandatory for this PR.

Reply:
Ok . I would make change in every file(including mine) regarding the goBackButton in the follow up. So that this PR and follow up PR would be very clear regarding it's aim and by then we can get the feedback from other testers too regarding the goBackButton